package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "oppia",
+  "version": "2.8.1",
+  "description": "Oppia enables the creation of interactive online lessons.",
+  "main": "gulpfile.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/oppia/oppia.git"
+  },
+  "author": "",
+  "license": "Apache v2",
+  "bugs": {
+    "url": "https://github.com/oppia/oppia/issues"
+  },
+  "homepage": "https://github.com/oppia/oppia#readme",
+  "devDependencies": {
+    "@mapbox/stylelint-processor-arbitrary-tags": "^0.2.0",
+    "@types/angular": "^1.6.54",
+    "@types/angular-animate": "^1.5.10",
+    "@types/angular-mocks": "^1.7.0",
+    "@types/ckeditor": "^4.9.2",
+    "@types/d3": "^3.5.40",
+    "@types/google.visualization": "0.0.46",
+    "@types/jasmine": "^3.3.8",
+    "@types/jasminewd2": "^2.0.6",
+    "@types/jquery": "^3.3.29",
+    "@types/jqueryui": "^1.12.1",
+    "@types/leaflet": "^1.4.0",
+    "@types/mathjax": "0.0.35",
+    "@types/mathjs": "^5.0.0",
+    "@types/mousetrap": "^1.6.1",
+    "@types/node": "^6.14.3",
```

Comment:
Hi @seanlip,
Is `node to 6.14.3` is left un-upgraded intentionally? we can use `10.14.6`

Reply:
Addressed in https://github.com/oppia/oppia/pull/6721